Source/Core/Common/ArmCPUDetect.cpp

```diff
@@ -69,7 +71,17 @@ void CPUInfo::Detect()
   CPU64bit = true;
   Mode64bit = true;
   vendor = CPUVendor::ARM;
+#ifdef __APPLE__
+  num_cores = std::thread::hardware_concurrency();
 
+  // M-series CPUs have all of these
+  bFP = true;
+  bASIMD = true;
+  bAES = true;
+  bSHA1 = true;
+  bSHA2 = true;
+  bCRC32 = true;
+#else
 #ifdef _WIN32
```

Comment:
consider perhaps `#elif defined(_WIN32)`

Reply:
Will update in next push.